scss/_variables.scss

```diff
@@ -528,6 +528,7 @@ $zindex-tooltip:            1070 !default;
 
 // Navs
 
+$nav-link-padding:              .5rem !default;
```

Comment:
I would call this $navbar-link-padding and put with others navbar-* variables.
Also consider if split between x/y axis wouldn't be better